Description: Upstream changes introduced in version 1.0.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 easy-format (1.0.0-1) unstable; urgency=low
 .
   * Initial release. (Closes: #605680)
 .
 The person named in the Author field signed this changelog entry.
Author: Sylvain Le Gall <gildor@debian.org>
Bug-Debian: http://bugs.debian.org/605680

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- easy-format-1.0.0.orig/Makefile
+++ easy-format-1.0.0/Makefile
@@ -24,6 +24,7 @@ simple_example.out: simple_example
 
 doc: ocamldoc/index.html easy_format_example.html
 ocamldoc/index.html: easy_format.mli
+	-mkdir ocamldoc
 	ocamldoc -d ocamldoc -html $<
 easy_format_example.html: simple_example.out simple_example.ml
 	cat simple_example.ml > easy_format_example.ml
@@ -37,6 +38,7 @@ soft-clean:
 	rm -f *.cm[iox] *.o *.annot \
 		test_easy_format lambda_example simple_example \
 		bytecode nativecode
+	-$(RM) -r ocamldoc
 
 clean: soft-clean
 	rm -f *.out ocamldoc/* \
